policy/release/cve.rego

```diff
@@ -203,7 +203,7 @@ _unpatched_vulnerabilities_deprecated := vulnerabilities if {
 
 _result_name := "SCAN_OUTPUT"
 
-_deprecated_result_name := "CLAIR_SCAN_RESULT"
+_deprecated_result_name := "SCAN_OUTPUT"
```

Comment:
I think we'd still want to keep this change as it would help the EC policies to catch if any of the outdated Clair tasks are still using the CLAIR_SCAN_RESULT result name.